tests/suites/test_suite_x509write.function

```diff
@@ -117,6 +138,9 @@ void x509_csr_check( char * key_file, char * cert_req_check_file, int md_type,
         TEST_ASSERT( mbedtls_x509write_csr_set_key_usage( &req, key_usage ) == 0 );
     if( set_cert_type != 0 )
         TEST_ASSERT( mbedtls_x509write_csr_set_ns_cert_type( &req, cert_type ) == 0 );
+    if ( set_extension != 0 )
+        TEST_ASSERT( csr_set_extended_key_usage( &req, MBEDTLS_OID_SERVER_AUTH,
+                            MBEDTLS_OID_SIZE(MBEDTLS_OID_SERVER_AUTH)) == 0);
```

Comment:
```suggestion
                            MBEDTLS_OID_SIZE( MBEDTLS_OID_SERVER_AUTH ) ) == 0 );
```

Reply:
General note about coding style: we'll soon be switching to a more standard style that will be automatically enforced, so I think until then we can be a bit more relaxed about it. I mean, it's good to try and keep it consistent it we can, but I don't think it would be worth reworking the PR if that was the only issue. (But there's a more important one with the magic number for a buffer size, thanks for catching it btw, which requires rework anyway, so, we can indeed fix the style while at it.)